I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 44, in script
    show emi_smirk Transform(xpos = 0.05, ypos = 0.95, anchor = (0.0, 1.0), zoom = 0.8)
Exception: Image 'emi_smirk' does not accept attributes 'Transform (xpos = 0.05, ypos = 0.95, anchor = (0.0, 1.0), zoom = 0.8)'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 44, in script
    show emi_smirk Transform(xpos = 0.05, ypos = 0.95, anchor = (0.0, 1.0), zoom = 0.8)
  File "C:\Users\anais\Downloads\renpy\renpy-8.1.1-sdk\renpy\ast.py", line 1111, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\anais\Downloads\renpy\renpy-8.1.1-sdk\renpy\ast.py", line 1074, in show_imspec
    renpy.config.show(name,
  File "C:\Users\anais\Downloads\renpy\renpy-8.1.1-sdk\renpy\exports\displayexports.py", line 477, in show
    if not base.find_target() and renpy.config.missing_show:
  File "C:\Users\anais\Downloads\renpy\renpy-8.1.1-sdk\renpy\display\image.py", line 421, in find_target
    self.target = target._duplicate(a)
  File "C:\Users\anais\Downloads\renpy\renpy-8.1.1-sdk\renpy\display\displayable.py", line 249, in _duplicate
    args.extraneous()
  File "C:\Users\anais\Downloads\renpy\renpy-8.1.1-sdk\renpy\display\displayable.py", line 109, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'emi_smirk' does not accept attributes 'Transform (xpos = 0.05, ypos = 0.95, anchor = (0.0, 1.0), zoom = 0.8)'.

Windows-10-10.0.22631 AMD64
Ren'Py 8.3.3.24100702+nightly
Emi VN 1.0
Mon Oct 21 19:36:01 2024
